abs.yaml

```diff
@@ -1,4 +1,5 @@
-upload_channels:
-  - sfe1ed40
 channels:
-  - sfe1ed40
+  - https://staging.continuum.io/prefect/fs/langchain-core-feedstock/pr6/e6e69e0
+  - https://staging.continuum.io/prefect/fs/langchain-text-splitters-feedstock/pr4/d2bb8ca
+  - https://staging.continuum.io/prefect/fs/langchain-feedstock/pr6/69eb35c
```

Comment:
```suggestion
```
This is what is causing the skips. The package exists in that channel.

Reply:
Thank you!